_i18n/en.yml

```diff
@@ -1365,6 +1365,24 @@ policies:
     We are happy to answer questions about our security and privacy practices.
     For more information, please visit [Help](site.baseurl/help)
     or [contact us](site.baseurl/contact).
+- section: Messaging terms and conditions
+  anchor: our-messaging-practices
+  content: |-
+    Login.gov uses SMS text messaging to help verify your identity when you create
+    a new account.  SMS text messages, or automated voice calls, may also be used if
+    you choose your telephone number as on of your authentication methods.
+
+    ### SMS Terms and Conditions
+    - Login.gov provides an authentication service and will not use text or
+      voice messaging for any other purpose.
+    - Messaging and data rates may apply. 
```

Comment:
Change to:

> Your carriers messaging and data rates may apply.

Reply:
That is better.  This is one where AWS was dictating the language, but I am going to change it and they can complain if they want.